pkg/operation/botanist/controlplane.go

```diff
@@ -280,29 +280,14 @@ func (b *Botanist) DeleteClusterAutoscaler() error {
 // to monitor the Shoot cluster whose control plane runs in the Seed cluster.
 func (b *Botanist) DeploySeedMonitoring() error {
 	var (
-		credentials      = b.Secrets["monitoring-ingress-credentials"]
-		basicAuth        = utils.CreateSHA1Secret(credentials.Data[secrets.DataKeyUserName], credentials.Data[secrets.DataKeyPassword])
-		alertManagerHost = b.Seed.GetIngressFQDN("a", b.Shoot.Info.Name, b.Garden.Project.Name)
-		grafanaHost      = b.Seed.GetIngressFQDN("g", b.Shoot.Info.Name, b.Garden.Project.Name)
-		prometheusHost   = b.ComputePrometheusIngressFQDN()
+		credentials       = b.Secrets["monitoring-ingress-credentials"]
+		credentialsOwners = b.Secrets["monitoring-ingress-credentials-owners"]
```

Comment:
Can we call that "users" (same of other occurrences)? I'll find `owners` a little misleading.

Reply:
Should all occurrences of `owners` be changed to `users`? Including the grafana deployment?

Reply:
Yes, that's what I would suggest - we normally call the people creating/using the shoot "users" and not "owners".